src/frontend/js/modules/box-overlay/box-overlay.spec.js

```diff
@@ -62,16 +62,15 @@ describe('The box-overlay Angular module', function() {
     return angular.element('.box-overlay-open');
   }
 
-  beforeEach(module('jadeTemplates'));
-
   beforeEach(function() {
     angular.mock.module('esn.box-overlay', function($provide) {
       esnI18nService = {
         translate: function(input) {
           return {
             toString: function() { return input; }
           };
-        }
+        },
+        getLocale: function() { return "fr" }
```

Comment:
Why do you have to add this, and:

```suggestion
        getLocale: function() { return 'fr'; }
```